src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -860,6 +866,25 @@ public void clickAddQuestionButton() {
         waitForPageToLoad();
     }
 
+    public void clickTemplateModalButton() {
+        addTemplateModalLoadButton.click();
+    }
+
+    public boolean isTemplateQuestionPanelExpanded(int qnNumber) {
+        // need to wait for the panel to be fully expanded before checking the class
+        waitForTextContainedInElementPresence(By.className("in"), Integer.toString(qnNumber));
```

Comment:
Same here. CSS selector should be more narrowed.

Reply:
Do you mean use `By.cssSelector()` instead of `By.className()`?